index.php

```diff
@@ -52,75 +52,82 @@ class="modal fade"
             </div>
-            <!-- Preloader -->
-            <div class="preloader" style="display:none;">
-                <img class="animation__shake" src="assets/img/rellutoolbox.svg" alt="rellutoolbox" width="600">
-            </div>
-        
-            <div class="content-wrapper" style="min-height: 768px;">
-            
-                <div class="content">
-                    <div class="container">
-                        <div class="row">
-                            <h1 class="text-center mt-3 pt-3 mb-3 pb-3">
-                                <img alt="logo" src="assets/img/rellutoolbox.svg" width="60%">
-                            </h1>
-                        </div>
-                        <div class="row">
-                            <?php
-                                $tools = json_decode(file_get_contents("./tools.json"), true);
-                                
-                                foreach($tools as $key => $value){
-                                    echo getTool(
-                                        $value["name"],
-                                        $value["displayname"],
-                                        $value["icon"],
-                                        $value["description"],
-                                        $value["bgclass"],
-                                        $value["content"]
-                                    );
-                                }
+        </nav>
+
+
+        <div class="content-wrapper" style="min-height: 768px;">
+
+            <div class="content">
+                <div class="container" style="margin-top: 80px;">
+                    <div class="row">
+
+                    </div>
+                    <div class="row">
+                        <?php
+                        $tools = json_decode(file_get_contents("./tools.json"), true);
+
+                        foreach ($tools as $key => $value) {
+                            echo getTool(
+                                $value["name"],
+                                $value["displayname"],
+                                $value["icon"],
+                                $value["description"],
+                                $value["bgclass"],
+                                $value["content"]
+                            );
+                        }
 
                         ?>
                     </div>
                 </div>
             </div>
         </div>
     </div>
-    <div class="floatingActionButton">
+    <!-- <div class="floatingActionButton">
         <div class="searchAnchor">
-            <input type="text" id="searchBox" class="searchBar" placeholder="Search Tools...">
+            <input type="text" id="searchBox" class="searchBar" placeholder="Wird zum Cookie Consenter later">
             <input class="searchSubmit" type="submit">
             <div class="searchToggle"></div>
         </div>
-    </div>
+    </div> -->
```

Comment:
Never Comment out Code for "later use" you have it in your git history and you can checkout a commit to get it back easily without dead code.